spring-integration-core/src/main/java/org/springframework/integration/transformer/EncodingPayloadTransformer.java

```diff
@@ -36,7 +36,7 @@ public EncodingPayloadTransformer(Codec codec) {
 	}
 
 	@Override
-	protected byte[] transformPayload(T payload) throws Exception {
+	protected byte[] transformPayload(T payload) throws Exception {//NOSONAR
```

Comment:
That's only a minor violation; I ignore minors; I want 0 critical and reduce major over time (introduce no new ones). I get an email whenever new major+ are introduced.

But, yes, I can disable this rule.